### ISSUE TITLE
font color is black

### ISSUE THREAD

**Issue by Yakarup** [StylishThemes/StackOverflow-Dark#245]
On default settings, except for the background image - Random Tile. I've reinstalled the style. Answer font colors are black. Post vote counts (scores) are black. Original post font color is black. Am using latest version v5.1.2

* **Browser**:  Google Chrome Version 117.0.5938.134 (Official Build) (64-bit)
* **Operating System**:  Windows 10 x64 Version 10.0.19045 Build 19045
* **Link to page with the issue**:  Everywhere across all stackexchange networks/sub-stackexchanges
* **Screenshot**: 
![Capture](https://github.com/StylishThemes/StackOverflow-Dark/assets/67720001/b9a12455-6d72-4a8f-82db-820695ecb900)

**Comment by SmartManoj** [comment]
dirty fix
```
p, .js-vote-count {
            color: #ddd !important;
}
```

**Comment by haxj** [comment]
Dirty fix, improved

```
p, .js-vote-count, body, pre, code {
    color: #ddd !important;
}
pre, code {
    background: #181818 !important;
 }
```

**Comment by muru** [comment]
Not a fix since this can't be done using user CSS, but using Greasemonkey or the like to add `theme-dark` to the `body` element's classes works remarkably well:

```js
document.getElementsByTagName('body')[0].classList.add('theme-dark')
```

It works so well that some problematic areas like diffs in the revision history of posts appear better than before. (the actual colours might not be to your taste, but at least they're readable now).

**Comment by Yakarup** [comment]
I also want to mention that only stackoverflow is unaffected. Otherwise, yeah, the font color is black on every stackexchange network.

**Comment by SylwesterZarebski** [comment]
As a quick fix I've inserted before line [67](https://github.com/StylishThemes/StackOverflow-Dark/blob/9eb02e3549156e8faab1c6427139394755671d15/stackoverflow-dark.user.css#L67):
```
		--theme-body-font-color: /*[[strong-color]]*/;
```

**Comment by SylwesterZarebski** [comment]
PR created. I've checked multiples SE sites, no more black text.

**Comment by mpeter50** [comment]
@SylwesterZarebski 's solution is already much better, but @muru 's solution also fixes the code blocks' background color!
Thank you both for your solutions!

**Comment by mpeter50** [comment]
Here is an example: https://superuser.com/questions/1613852/how-do-i-run-a-sudo-command-on-a-remote-machine-using-ssh

This is how it looks for me with your fix (no userscript, repo's userstyle edited by adding `--theme-body-font-color: /*[[strong-color]]*/;` after line 66):

![image](https://github.com/StylishThemes/StackOverflow-Dark/assets/83356418/f3b37f1f-00c5-4d07-8948-e48e03f33297)

This is how it looks without editing the userstyle, but with the userscript:

![image](https://github.com/StylishThemes/StackOverflow-Dark/assets/83356418/b8765b1e-cc8f-4edd-ab10-8716a8d5dd0b)

**Comment by muru** [comment]
@mpeter50 it looks like you're not using [the StackOverflow Syntax Themes usercss](https://github.com/StylishThemes/Feature-Override-Styles/blob/master/stackoverflow-syntax-themes.user.css). Check [the README](https://github.com/StylishThemes/StackOverflow-Dark#readme), it's "a __required additional style__".

**Comment by mpeter50** [comment]
Oh, thats right, thanks for pointing that out! I have now installed that style and this way its ok with [this](https://github.com/StylishThemes/StackOverflow-Dark/issues/245#issuecomment-1777809032) modification, without a userscript.

This way inline code blocks are less visually distinct than before, but after tweaking the settings of the syntax style a little its better then before! Just jotting it down in case anyone will find it useful.

**Comment by Yakarup** [comment]
Bug fixed by commit e53dd43f3c1a52921f39234405f594b246f61f17

**Comment by Yakarup** [comment]
Bug fixed by commit e53dd43f3c1a52921f39234405f594b246f61f17
